sharding-scaling/sharding-scaling-core/src/test/java/org/apache/shardingsphere/shardingscaling/core/web/HttpServerHandlerTest.java

```diff
@@ -66,31 +66,31 @@ public void setUp() {
 
     @Test
     public void channelReadStartFailed() {
```

Comment:
Sorry, I forgot review that unit test method name should start with `assert`
For example this test method should change to `assertChannelReadStartFailed()`;